user-manual/metadata/METS.rst

```diff
@@ -0,0 +1,182 @@
+.. _METS_schema:
+
+======================
+METS in Archivematica
+======================
+
+General description
+--------------------
+
+The Metadata Encoding & Tranmission Standard, or `METS`_, is a schema for
+encoding various metadata, expressed in XML . It essentially acts a wrapper
+around other metadata standards, such as PREMIS and Dublin Core that are used in
+Archivematica.
+
+The METS file in Archivematica will have a basic generic structure that is
+comprised of the following sections:
+
+* metsHdr (METS header) (1 only)
```

Comment:
This is nice, as I skipped below I was thinking about expansion of these terms and then skipped back to the top and saw them!!! 

On the syntax `(1 only)` what do you think of the academic approach to numerals? I believe it's 1 to 9 are written one to nine, and then after 10 digits. I don't know our approach across the larger body of docs. I don't think we have it in the style guide at all [ref](https://github.com/artefactual/archivematica-docs/blob/1.11/style_guide.md).

Reply:
I will attempt to go for the Purdue OWL's recommendation then as I edit. Maybe we can consider putting this in the wiki style guide?

Reply:
FYI it's now in the style guide! https://github.com/artefactual/archivematica-docs/blob/1.11/style_guide.md#formatting-and-punctuation

Reply:
Love this @sallain! Thanks!